zokrates_fs_resolver/src/lib.rs

```diff
@@ -83,14 +76,14 @@ mod tests {
 
     #[test]
     fn non_existing_file() {
-        let fs_resolver = FileSystemResolver::new();
+        let fs_resolver = FileSystemResolver::new("");
```

Comment:
can we rename `new` to `with_stdlib_root` to make it clearer?

Reply:
yeah good catch